### PR TITLE
New documentation on IPythonMagics and getting started

### PR DESCRIPTION
**Pull request by a-larkins99**
## Content:
Adds both:
- GettingStarted.md
- IPythonMagics_commands.md

These add new guidance for setting up and starting with pace-python using Horace 3.6.3, as well as documentation on the new IPythonMagics commands following their migration to libpymcr. 

**Note:**  GettingStarted.md refers to pace-python `version: 0.3.0`. I have made the assumption that this is what the release will be called. Please update this as needed.

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+# Pace-Neutrons: getting started
+version: 0.3.0
+## Introduction
+
+Pace-Neutrons, previously known as pace-python, is a python package which brings Horace 3.6.3 and SpinW to python removing the need for Matlab whilst keeping the commands you have become familiar with.
```
Comment (oerc0122) [inline docs/GettingStarted.md:5]:
```suggestion
Pace-Neutrons (formerly known as pace-python) is a Python package which brings Horace >3.6.3 and SpinW to Python, removing the need for Matlab whilst keeping the commands you have become familiar with.
```

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+## Requirements
+
+- Windows
+- Python 3.8, 3.9 or 3.10 (You can check this by running `python --version` in your terminal)
+- Matlab R2021b runtime
+
+## Installation
+
+The easiest way to install `pace-neutrons` and its installer is by downloading the `pace_neutrons_installer` from the [release](https://github.com/pace-neutrons/pace-python/releases) on github.
+When this is run, it will install the pace-neutrons pypi package as well as the Matlab runtime required to use the package.
```
Comment (oerc0122) [inline docs/GettingStarted.md:16]: Link to PyPI package

File: docs/GettingStarted.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+This will require you to download and install the corresponding Matlab runtime (by default R2021b) which can be found along with installation instructions 
+[here](https://uk.mathworks.com/products/compiler/matlab-runtime.html). The correct runtime can also be obtained by running the `pace_neutrons --install-mcr`
+command in your terminal. 
+
+## Usage
+
+### Differences to regular Horace
+
+All of the familiar [Horace commands](https://pace-neutrons.github.io/Horace/v3.6.3/) can be used with pace-neutrons simply by prefixing them 
+with `m.` and dropping the semi-colon at the end of the line. For example,
```
Comment (oerc0122) [inline docs/GettingStarted.md:31]:
```suggestion
with `m.`, transforming MATLAB arrays and cell-arrays to numpy `array`s/`list`s and `struct`s to `dict`s and dropping the semi-colon at the end of the line. For example,  
```

File: docs/GettingStarted.md (hunk 2 above)
Comment (a-larkins99) [reply docs/GettingStarted.md:16]: We haven't done the full release yet. Duc wanted to wait until Horace 4 is released I believe.

File: docs/GettingStarted.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+- **HORACE_PATH** - Provide a path to your version of Horace
+- **SPINW_PATH**  - Provide a path to your version of SpinW
+- **HORACE_VERSION** - Set the Horace release version to download e.g. 3.6.3
+- **SPINW_VERSION** - Set the SpinW version (git tag/branch/hash)
+- **SPINW_REPO** - Set the repo to get SpinW from e.g. "mducle/spinw"
+- **Matlab_ROOT_DIR** - Provide a path to desired Matlab version
+
+#### Example
+
+`python setup.py bdist_wheel -DWITH_SPINW=OFF -DHORACE_PATH="/path/to/your/systems/Horace"`
```
Comment (oerc0122) [inline docs/GettingStarted.md:227]: Probably wants to be triple-quoted block

File: docs/GettingStarted.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+selecting the `Download ZIP` option. This demo is complete with a demo `.py` script as well as a `.ipynb` 
+file for use in a Jupyter notebook.
+
+
+#### Horace GUI
+
+The Horace GUI can be opened, following the setup procedure outlined above, by 
+running `m.horace()`
+
+## Jupyter notebook specifics
```
Comment (oerc0122) [inline docs/GettingStarted.md:199]: Link to other doc on magic

File: docs/GettingStarted.md (hunk 2 above)
Comment (oerc0122) [reply docs/GettingStarted.md:16]: Doesn't matter too much, these docs will outlast this release.

**Review by mducle — APPROVED** [review]
Thanks. Looks good to me, ok to merge after addressing Jacob's comments.

File: docs/GettingStarted.md (hunk 2 above)
Comment (a-larkins99) [reply docs/GettingStarted.md:16]: I can add the main pypi page then.

File: docs/GettingStarted.md (hunk 5 above)
Comment (a-larkins99) [reply docs/GettingStarted.md:199]: How do you link to another doc? I don't know what the url will be to the thing once it is merged

Comment (a-larkins99) [reply docs/GettingStarted.md:199]: @oerc0122

**Review by oerc0122 — APPROVED** [review]

**Review by mducle — APPROVED** [review]

**Pull request merged**